docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -33,33 +33,11 @@ ref over User1, Orderer
     see fpc-lifecycle-v2.puml
 end ref
 
-note right ECC3 #FFaa00
-    How can we express that we need an enclave here?
-    In particular, an enclave that has ECC_MR and also has a valid attestation?!
-end note
-
 ref over User1, Orderer
     register enclaves and key-dist
     see fpc-registration.puml and fpc-key-dist.puml
 end ref
```

Comment:
I cannot add a suggestion as it is not in your diff, but we should mention designated peer endorsement policy as 2 of 3 actually doesn't work with the plugin approach (or we also need deterministic signatures and alike ...)

Actually, might be worth mention this limitation, this is only for MVP and we have architecture via MSP to go beyond?

Reply:
Ok added a comment